readme.md

```diff
@@ -163,6 +163,30 @@ Once the application has been created via `eb create ...` all subsequent updates
 eb deploy ${environment name} --profile ${your AWS profile}
 ```
 
+#### Switching Loadbalancer settings between accounts
+
+Loadbalancer settings is required for HTTPS to run behind the load balancers for Elastic Beanstalk. Each Loadbalancer setting is unique to each AWS profile, so switching configuration files is required to deploy to different AWS profiles.
```

Comment:
...settings are required...

...is required when deploying to different...

Reply:
Much more clear now!!!! Now I know why we need this.